PSHTML/Functions/Public/sub.ps1

```diff
@@ -1,89 +1,81 @@
 Function SUB {
     <#
     .SYNOPSIS
-    Create a SUB title in an HTML document.
-
+        Create a SUB tag in an HTML document.
+    .DESCRIPTION
+        Create a SUB tag in an HTML document. 
```

Comment:
Could you please update the description with what the <sub> tag actually offers?
You can get it from here https://www.w3schools.com/tags/tag_sub.asp